<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JudgeWidget</class>
 <widget class="QWidget" name="JudgeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Judge</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="advice">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>推荐的程序对：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="8" column="1">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="7" column="1">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="open">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>使用地址栏打开文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>请判断以下两个程序是否等价：</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="yes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>等价</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>3700</width>
       <height>2350</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QPushButton" name="doubt">
     <property name="text">
      <string>存疑</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="no">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>不等价</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>3700</width>
       <height>2350</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="OJfile">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
